polars/polars-arrow/src/utils.rs

```diff
@@ -154,16 +154,6 @@ impl<T: NativeType> FromTrustedLenIterator<Option<T>> for PrimitiveArray<T> {
     }
 }
 
-impl<T: NativeType> FromTrustedLenIterator<T> for PrimitiveArray<T> {
```

Comment:
Why is this removed?

Reply:
I'm so sorry that was a mistake. I was trying to for some time to implement trusted collect for Result next to there, but deleted it again. I did not the check the diff well enough before committing.